client/src/pages/CurrentUserProfile/CurrentUserProfile.jsx

```diff
@@ -31,18 +31,83 @@ const CurrentUserProfile = () => {
       if (response.ok) {
         setUser(null);
         navigate("/login");
+      }
+    } catch (error) {
+      setError(error.message);
+    }
+  };
+
+  // fetch user profile picture from backend
+  const fetchUserProfilePicture = async (
```

Comment:
Discussed offline - we agreed that this wasn't necessary, and you could just put the URL straight into the image tag's `src` attribute.